Update stack to 7.9.2

anyasabo opened the pull request:
Also updates the `hack/update-samples-stack-version.sh` script to cover a few more files. We could also extend this to cover `hack/operatorhub/config.yaml` but I opted to just leave a note about it for now

thbkrkr commented:
We've missed some versions in the [latest](https://github.com/elastic/cloud-on-k8s/pull/3642) [updates](https://github.com/elastic/cloud-on-k8s/pull/3610). This stack version update is definitively very prone to oversights.

To be complete:
```sh
> ag '7.[89].[01]' --ignore pkg --ignore go.sum --ignore hack/release-notes --ignore docs --ignore "*_test.go" --ignore NOTICE.txt --ignore stream.json --ignore builder.go

test/e2e/test/version.go
19:     LatestVersion7x = "7.9.0" // version to synchronize with the latest release of the Elastic Stack

test/e2e/kb/testdata/kibana_standalone.yaml
7:  version: 7.8.1
21:  version: 7.8.1

config/recipes/beats/auditbeat_hosts.yaml
79:        #  image: docker.elastic.co/beats/auditbeat:7.8.1

Makefile
377:E2E_STACK_VERSION          ?= 7.9.0

hack/upgrade-test-harness/conf.yaml
16:    stackVersion: 7.8.0
20:    stackVersion: 7.8.0

hack/upgrade-test-harness/testdata/upcoming/stack.yaml
7:  version: 7.8.1
19:  version: 7.8.1
29:  version: 7.8.1

hack/upgrade-test-harness/testdata/v112/stack.yaml
7:  version: 7.8.1
19:  version: 7.8.1
29:  version: 7.8.1
```

I created https://github.com/elastic/cloud-on-k8s/issues/3612 for that purpose. Maybe we should just be more declarative in the script and not just rely on a `find`. It's a small task but I feel like we waste a few minutes every month. The Elastic release train is going fast :]

anyasabo commented:
Good catch @thbkrkr . I'm not sure the `hack/upgrade-test-harness` is easily update-able and is only manually run anyway, so I left that out of this. I manually updated `Makefile`, `config/recipes/beats/auditbeat_hosts.yaml`, and `test/e2e/test/version.go`. I added `test/e2e` to the paths to search. To get those three I updated manually I think we would need to change our approach as you mentioned.

I also wonder if just outputting the `ag`/`grep` search at the end of the auto-update script would be a step forward to at least let people know maybe there's something that still needs attention.
